packages/plugins/src/foundry/utils/index.ts

```diff
@@ -51,7 +51,7 @@ export const getContractArtifact = async (
 
   if (!(await existsAsync(completeFilePath))) {
     throw new Error(
-      `Could not find artifact for: ${name}. Did you forget to import it in your script file?`
+      `Could not find artifact for: ${name}. Are you sure you've properly configured your src, script, and test directories in your foundry.toml file?`
```

Comment:
This sounds a bit vague to me. Maybe something like: `Could not find artifact for: ${name}. Please make sure that this contract exists in either the src, script, or test file that you've configured in your foundry.toml.`